lib/containers/runtime.pm

```diff
@@ -32,27 +34,73 @@ sub _rt_script_output {
     return script_output($self->runtime . " " . $cmd, @args);
 }
 
-=head2 build
+=head2 create_container($image, $name, [%args])
 
-Build a container.
+Creates a container.
+C<image> the name of the image we create the container from.
+C<name> is the given name of the created container.
+C<args> can be anything you want to run in the container, similar to C<run>. for 
+instance: C<docker create -it tumbleweed bash>
+
+=cut
+sub create_container {
+    my ($self, $image, $name, $args) = @_;
```

Comment:
As in some previous comment, I would suggest all the input parameters to be in this way: 
```suggestion
    my ($self, %args) = @_;
```
Then you use the values with `$args{image}`, `$args{name}`.
And if you want to make some parameter mandatory, just do 
`die('Must provide an image') if (!$args{image});`